### ISSUE TITLE
Paste JSON directly into GUI?

### ISSUE THREAD

**Issue by edeutsch** [RTXteam/RTX#852]
As we work with the ARS more (who will be sending us JSON query_graphs), it would be handy for testing to be able to also paste JSON directly into the UI. I'm imagining a 4th tab next to DSL that is called [JSON]. In it is a textarea like for the DSL. And here the user can paste in something like:
```
    "query_graph": {
      "edges": [
        {
          "id": "qg2",
          "source_id": "qg1",
          "target_id": "qg0"
        }
      ],
      "nodes": [
        {
          "id": "qg0",
          "curie": "CHEMBL.COMPOUND:CHEMBL112"
        },
        {
          "id": "qg1",
          "type": "protein"
        }
      ]
    }
```
And when pressing the POST button, the UI would post that query_graph (in a message) to the endpoint (after potentially catching parsing errors)

Making it all integrate with the other modes would be a nice touch for rev2, but not needed for rev1.

One of the scenarios I'm thinking of is Mark Williams sending me a JSON snippet like the above and asking "why doesn't this work?" Would make it easiest to test if I could paste it into a GUI window and click POST and see what happens and sleuth from there.

**Comment by edeutsch** [comment]
(a button to pre-populate it with the above example would be nice, too)

**Comment by isbluis** [comment]
Added the 4th tab input.

Do note that one should only enter the contents of the `query_graph` element (i.e. omit `"query_graph":`  from the above example).

**Comment by edeutsch** [comment]
Looks great. Would you also add an [EXAMPLE] button, that pastes in something like this:
```
{
      "edges": [
        {
          "id": "qg2",
          "source_id": "qg1",
          "target_id": "qg0"
        }
      ],
      "nodes": [
        {
          "id": "qg0",
          "curie": "CHEMBL.COMPOUND:CHEMBL112"
        },
        {
          "id": "qg1",
          "type": "protein"
        }
      ]
    }
```

**Comment by edeutsch** [comment]
Maybe also useful while you're at it to have a DSL example for the DSL tab:
```
add_qnode(name=acetaminophen, id=n0)
add_qnode(type=protein, id=n1)
add_qedge(source_id=n0, target_id=n1, id=e0)
expand(edge_id=e0)
overlay(action=compute_ngd, virtual_relation_label=N1, source_qnode_id=n0, target_qnode_id=n1)
resultify()
```

**Comment by isbluis** [comment]
Examples added.

**Comment by isbluis** [comment]
@edeutsch : Once this is approved and pushed to production, should the text on this page be updated to reflect this 4th way of querying our tools?

https://arax.rtx.ai/overview.html

**Comment by edeutsch** [comment]
it certainly should!

now done and pushed to master.
